src/http.rs

```diff
@@ -19,7 +19,13 @@ impl Default for HttpClient {
     fn default() -> Self {
         HttpClient {
             client: reqwest::blocking::Client::builder()
-                .connect_timeout(Duration::new(10, 0))
+                // Keep idle connections in the pool for up to 55s. AWS
+                // ALBs will drop idle connections after 60s and the default
```

Comment:
nit: took me a second to realize `ALB` was Application Load Balancer, mind just using the whole phrase here instead?